postings.go

```diff
@@ -240,6 +241,40 @@ func (it *listPostings) Err() error {
 	return nil
 }
 
+type bytePostings struct {
```

Comment:
This should probably be named `bigEndianPostings` to be more specific about the contents of the byte slice.
Later, we might want to improve further by using compression schemes on the postings. It will still be a `bytePostings` but with a different encoding scheme.